packages/fiori/src/UploadCollection.js

```diff
@@ -125,6 +125,16 @@ const metadata = {
 		},
 	},
 	events: /** @lends sap.ui.webcomponents.fiori.UploadCollection.prototype */ {
+		/**
+		 * Fired when an element is dropped inside the drag and drop overlay.
```

Comment:
Normally we don't document browser events, because they are fired and work anyway. However, we do this for browser events that we modify (such as this one) in order to explain what is different than the expected normal behavior. What you described here is correct, but it would have been correct before the change too. So I'd suggest adding:

`<b>Note:</b> The <code>drop</code> event is fired only for the drag and drop overlay and ignored for the other parts of the <code>ui5-upload-collection</code>`

or something to this effect.

Reply:
Done.